app.js

```diff
@@ -6,6 +6,12 @@ export const getArticlesData = () => {
     })
 }
 
+export const getSpecficArticleData = (article_id) => {
+    return axios.get(`https://nc-news-f67l.onrender.com/api/articles/${article_id}`).then(({data}) => {
+        return data
+    })
+}
```

Comment:
Nice, but instead of just sending the whole data object, you can send the property of article which is exactly the value you want on the other side of this promise chain